### PR TITLE
Improve documentation of use with Podman

### PR DESCRIPTION
**Pull request by rhatdan**
Replace: https://github.com/containers/dnsname/pull/39
Originally submitted by @omarc1492

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@voigt @TomSweeneyRedHat @omarc1492 @baude @mheon @vrothberg PTAL

File: README_PODMAN.md [hunk 1]
```diff
@@ -17,25 +17,46 @@ Using your package manager, install the *dnsmasq* package.  For Fedora, this wou
...
+
+ ```
+{
+   "cniVersion": "0.4.0",
+   "name": "foobar",
+   "plugins": [
+      ...
+      {
+	 "type": "dnsname",
+	 "domainName": "dns.podman"
```
Comment (baude) [inline README_PODMAN.md:41]: "capabilities": {
            "aliases": true
         }

add this to the stanza

**Comment by mheon** [comment]
LGTM

**Pull request merged**